roles/lib_openshift/src/test/integration/oc_process.yml

```diff
@@ -66,13 +64,14 @@
       reconcile: false
     register: procout
 
+  # These assertions could fail due to changes in item order
   - assert:
       that:
       - not procout.changed
-      - procout.results.results['items'][0]['metadata']['name'] == 'testdb'
-      - procout.results.results['items'][0]['kind'] == 'Service'
-      - procout.results.results['items'][1]['metadata']['name'] == 'testdb'
-      - procout.results.results['items'][1]['kind'] == 'DeploymentConfig'
+      - procout.module_results.results['items'][0]['metadata']['name'] == 'testdb'
+      - procout.module_results.results['items'][0]['kind'] == 'Secret'
```

Comment:
Its not clear why kind: Secret is being used here

Reply:
These integration tests had not been run in a long time.  The order of the results being returned changed from having 'Service' first to having 'Secret' returned.  This is only changing so items are in the order they are returned from the current version of the product, although it is not a 'failure' if they are in a different order.  This is why I added the comment on line 67.